src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Subscriber.kt

```diff
@@ -65,6 +68,8 @@ fun subscribe(subscription: String, receiver: CompilerFleetMessageReceiver) {
     }
 }
 
+val config = ConfigFactory.load()
```

Comment:
Why is it declared here while used in other file?
Also, given that Typesafe Config object is immutable, you may want to pass Config to your `compile` function rather than accessing a singleton. This will allow for easier testing.

Reply:
> Why is it declared here while used in other file?
Thought, it's appropriate to declare at the program start point

What's the reason to pass Config to `compile`?
The only thing, that I can do with config is obtaining `compileCommand` value, but it's already a parameter, which can be replaced in test cases. It's more straightforward, than mocking config, no?

Reply:
Done